nilearn/plotting/surface/tests/test_matplotlib_backend.py

```diff
@@ -195,3 +203,23 @@ def test_compute_facecolors_matplotlib_deprecation():
             0.5,
             alpha,
         )
+
+
+@pytest.fixture
+def engine():
+    return ENGINE
+
+
+class TestPlotlyBackend(BaseTestSurfaceBackend):
+    """Extend BaseTestSurfaceBackend class to run all test methods for
+    'matplotlib' engine.
+
+    This class should override the test methods of BaseTestSurfaceBackend if
+    the behavior for 'matplotlib' engine differs from the behavior expected in
+    BaseTestSurfaceBackend.
+
+    Otherwise all methods will be run injecting the engine fixture defined in
+    this module.
```

Comment:
so you are saying that even though there is a pass here, this will actually run all the tests in the base class

that was not at all obvious to me

Reply:
I see

Reply:
For refactoring this function, it was not really necessary. It is necessary when the implementation differs. For example, for the functions which are not implemented in plotly or the returned object is tested differently. 
Otherwise, it would be ok to write all tests in `test_backend` and parameterize with `engine`.

Reply:
> It is necessary when the implementation differs for backends.
> For example, for the functions which are not implemented in plotly.

yup I was comparing with your longer PR and suspected that this was the case